[2025-07-06T11:32:31.336+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-06T11:32:31.348+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: practice_etl_pipeline.extract manual__2025-07-06T11:32:26.527736+00:00 [queued]>
[2025-07-06T11:32:31.352+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: practice_etl_pipeline.extract manual__2025-07-06T11:32:26.527736+00:00 [queued]>
[2025-07-06T11:32:31.352+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-06T11:32:31.360+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract> on 2025-07-06 11:32:26.527736+00:00
[2025-07-06T11:32:31.363+0000] {clientserver.py:570} INFO - Closing down clientserver connection
[2025-07-06T11:32:31.364+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=249) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-06T11:32:31.367+0000] {standard_task_runner.py:63} INFO - Started process 388 to run task
[2025-07-06T11:32:31.368+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'practice_etl_pipeline', 'extract', 'manual__2025-07-06T11:32:26.527736+00:00', '--job-id', '261', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp8qfp9v41']
[2025-07-06T11:32:31.372+0000] {standard_task_runner.py:91} INFO - Job 261: Subtask extract
[2025-07-06T11:32:31.405+0000] {task_command.py:426} INFO - Running <TaskInstance: practice_etl_pipeline.extract manual__2025-07-06T11:32:26.527736+00:00 [running]> on host 93b0937d74ff
[2025-07-06T11:32:31.457+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='practice_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-06T11:32:26.527736+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-06T11:32:26.527736+00:00'
[2025-07-06T11:32:31.459+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-06T11:32:32.881+0000] {logging_mixin.py:188} INFO - +----------+------+--------------------------------------+-------+
|timestamp |status|message                               |obj_num|
+----------+------+--------------------------------------+-------+
|2025-07-06|ИНФО  |файлы обработаны и переданы в загрузку|6      |
+----------+------+--------------------------------------+-------+
[2025-07-06T11:32:32.886+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-06T11:32:32.887+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-06T11:32:32.892+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=practice_etl_pipeline, task_id=extract, run_id=manual__2025-07-06T11:32:26.527736+00:00, execution_date=20250706T113226, start_date=20250706T113231, end_date=20250706T113232
[2025-07-06T11:32:32.905+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-06T11:32:32.915+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-06T11:32:32.918+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
